app/config/index.js

```diff
@@ -29,7 +32,10 @@ export const getConfig = () => {
     evidenceReviewTemplateId: process.env.EVIDENCE_REVIEW_TEMPLATE_ID,
     emailReplyToId: process.env.EMAIL_REPLY_TO_ID,
     evidenceFollowUpTemplateId: process.env.EVIDENCE_FOLLOW_UP_TEMPLATE_ID,
-    sfdRequestMsgType: 'uk.gov.ffc.ahwr.sfd.request'
+    sfdRequestMsgType: 'uk.gov.ffc.ahwr.sfd.request',
+    multiHerds: {
+      enabled: process.env.MULTI_HERDS_ENABLED === 'true'
```

Comment:
is MULTI_HERDS_ENABLED set up in the helm values file? I think it needs adding `helm/ffc-ahwr-message-generator/values.yaml`